src/stac_iitmandi/views.py

```diff
@@ -1,7 +1,30 @@
 from django.shortcuts import render
 
-def home(request):
-    return render(request, 'stac_iitmandi/home.html')
 
+# rendering home page 
+def home(request):
+    return render(request, 'stac_iitmandi/home.html' , {'title': 'Home'})
+# rendering template page
 def template(request):
-    return render(request, 'stac_iitmandi/template.html')
\ No newline at end of file
+    return render(request, 'stac_iitmandi/template.html')
+# rendering ASTRAX page
+def astrax(request):
+    return render(request, 'stac_iitmandi/astrax.html', {'title': 'Astrax'})
+# rendering Utkarsh page
+def utkarsh(request):
+    return render(request, 'stac_iitmandi/utkarsh.html', {'title': 'Utkarsh'})
+# rendering Zenith page
+def zenith(request):
+    return render(request, 'stac_iitmandi/zenith.html', {'title': 'Zenith'})
+# rendering Alumni's page    
+def alumni_s(request):
+    return render(request, "stac_iitmandi/alumni_s.html", {'title': "Alumni's"})
```

Comment:
I think `alumni_s` could be changed to `alumni`, and maybe the `Alumni's` to `Alumni` in navbar and here too.

Reply:
Ok will do that.